plugins/uefi-capsule/fu-uefi-bootmgr.c

```diff
@@ -249,6 +249,147 @@ fu_uefi_setup_bootnext_with_loadopt(FuEfiLoadOption *loadopt, GError **error)
 	return TRUE;
 }
 
+static GHashTable *
+fu_uefi_bootmgr_parse_csv(const gchar *data, size_t size, GError **error)
+{
+	g_auto(GStrv) lines = NULL;
+	g_autoptr(GHashTable) hash = NULL;
+	g_autofree gchar *nt_data = NULL;
+
+	hash = g_hash_table_new_full(g_str_hash, g_str_equal, g_free, g_free);
+
+	/* null terminate the file */
+	nt_data = g_strndup(data, size);
+
+	lines = g_strsplit(nt_data, "\n", -1);
```

Comment:
you can make this a bit simpler with fu_strsplit

Reply:
Done.